torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -544,6 +573,13 @@ std::shared_ptr<SugaredValue> toSugaredValue(
 
   py::bool_ isFunction = py::module::import("inspect").attr("isfunction")(obj);
   if (py::cast<bool>(isFunction)) {
+    auto overloads =
```

Comment:
Looks like the infrastructure for overloaded functions is only implemented for Python. Do we have a plan for resolving overloaded functions defined from the string frontend (for example, how will we serialize overloaded functions)?

Reply:
Any overloaded function that needs to be serialized will already be instantiated in one of its overloads, that function gets serialized normally.